picard/track.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, id, album=None):
     def __repr__(self):
         return '<Track %s %r>' % (self.id, self.metadata["title"])
 
+    def __lt__(self, other):
```

Comment:
Wouldn't it make sense to implement other methods of the same family for consistency ?
```
  def __eq__(self, other):
  def __ne__(self, other):
  def __gt__(self, other):
  def __ge__(self, other):
  def __le__(self, other):
```